BierBuddy.Core/MyBuddies.cs

```diff
@@ -41,5 +41,10 @@ public void HandleAppointment(Appointment appointment, bool accepted)
                 _DataAccess.DeclineAppointment(appointment.AppointmentID);
             }
         }
+
+        public string visitorName(long visitorID)
```

Comment:
1. Dit is geen logische functienaam, graag een Get ervoor zodat het duidelijk is wat het doet.
2. Functienamen altijd beginnen met een hoofdletter, dus VisitorName.